samples/snippets/evaluate_processor_version_sample.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# [START documentai_evaluate_processor_version]
+
+from google.api_core.client_options import ClientOptions
+from google.api_core.exceptions import NotFound
+from google.cloud import documentai_v1beta3 as documentai
+
+# TODO(developer): Uncomment these variables before running the sample.
+# project_id = 'YOUR_PROJECT_ID'
+# location = 'YOUR_PROCESSOR_LOCATION' # Format is 'us' or 'eu'
+# processor_id = 'YOUR_PROCESSOR_ID'
+# processor_version_id = 'YOUR_PROCESSOR_VERSION_ID'
+# gcs_input_uri = # Format: gs://bucket/directory/
+
+
+def evaluate_processor_version_sample(
+    project_id: str,
+    location: str,
+    processor_id: str,
+    processor_version_id: str,
+    gcs_input_uri: str,
+):
+    # You must set the api_endpoint if you use a location other than 'us', e.g.:
+    opts = ClientOptions(api_endpoint=f"{location}-documentai.googleapis.com")
```

Comment:
NIT: You can also set `ClientOptions` with a dictionary. This is clearer as to what you're doing, so it's fine to leave as-is.